base/Base_compiler.jl

```diff
@@ -158,8 +158,12 @@ end
 """
     time_ns() -> UInt64
 
-Get the time in nanoseconds relative to some arbitrary time in the past. The primary use is for measuring the elapsed time
-between two moments in time.
+Get the time in nanoseconds relative to some machine-specific arbitrary time in the past.
+The primary use is for measuring elapsed times during program execution.  The return value is guaranteed to
+be monotonic (mod 2⁶⁴) while the system is running, and is unaffected by clock drift or changes to local calendar time,
+but it may change arbitrarily across system reboots or suspensions.
+
+(Although the returned time is always in nanoseconds, the timing resolution is platform-dependent.)
```

Comment:
Why is this in its own paragraph with parenthesis? Seems like it is just as relevant as the other text.

Reply:
Because I thought it was kind of obvious that units ≠ resolution (i.e. precision ≠ accuracy), so this paragraph is just a reminder?